bedrock/base/templates/base-resp.html

```diff
@@ -167,13 +167,36 @@
     </div><!-- close #outer-wrapper -->
 
     <!--[if IE 9]>
-      <script src="{{ static('js/libs/matchMedia.js') }}"></script>
+      {% javascript 'matchmedia' %}
     <![endif]-->
 
+    {#
+      The logic mess below will be simplified when all pages are refactored
+      to behave well in old IE without page-specific JS.
+    #}
+    {% if force_modern_js != True %}
```

Comment:
If this means we can avoid the conditional then I'm all for making the `/contact/` pages look ok without JS. We could retire the oldIE JS bundle for those pages 👍